src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Yes I agree, it's useless. Do you want to make `CalendarDay` a class so we can pass `this`, or do you want no `target` argument for the `onClick` function?